.github/workflows/prod-container.yml

```diff
@@ -120,3 +120,17 @@ jobs:
                   task-definition: ${{ steps.task-def-plugins.outputs.task-definition }}
                   service: posthog-production-plugins
                   cluster: posthog-production-cluster
+    sentry:
+        name: Notify Sentry of a production release
+        runs-on: ubuntu-20.04
+        steps:
+            - name: Checkout master
+              uses: actions/checkout@v2
+            - name: Notify Sentry
+              uses: getsentry/action-release@v1
+              env:
+                  SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+                  SENTRY_ORG: posthog
+                  SENTRY_PROJECT: posthog
+              with:
```

Comment:
As per the defaults here:

https://github.com/getsentry/action-release#parameters

Seems as though this will tag releases based on the commit hash, which is fine, but not aligned with our version naming system... Could we pull our version name into this?

Reply:
We should be able to pull this from `version.py` or the latest tag

Reply:
This is for Cloud only though, right? In which case version doesn't matter at all, only the commit

Reply:
ah right

Reply:
Exactly! Let me know if we can move forward with this. FYI relevant secret already configured for the repo.